src/Microsoft.ServiceFabric.Client.Http/Generated/Serialization/NameRepartitionSchemeDescriptionConverter.cs

```diff
@@ -0,0 +1,85 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.ServiceFabric.Client.Http.Serialization
+{
+    using System;
+    using System.Collections.Generic;
+    using Microsoft.ServiceFabric.Common;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Linq;
+
+    /// <summary>
+    /// Converter for <see cref="NameRepartitionSchemeDescription" />.
+    /// </summary>
+    internal class NameRepartitionSchemeDescriptionConverter
+    {
+        /// <summary>
+        /// Deserializes the JSON representation of the object.
+        /// </summary>
+        /// <param name="reader">The <see cref="T: Newtonsoft.Json.JsonReader" /> to read from.</param>
+        /// <returns>The object Value.</returns>
+        internal static NameRepartitionSchemeDescription Deserialize(JsonReader reader)
+        {
+            return reader.Deserialize(GetFromJsonProperties);
+        }
+
+        /// <summary>
+        /// Gets the object from Json properties.
+        /// </summary>
+        /// <param name="reader">The <see cref="T: Newtonsoft.Json.JsonReader" /> to read from, reader must be placed at first property.</param>
+        /// <returns>The object Value.</returns>
+        internal static NameRepartitionSchemeDescription GetFromJsonProperties(JsonReader reader)
+        {
+            var namesToAdd = default(IEnumerable<string>);
+            var namesToRemove = default(IEnumerable<string>);
+
+            do
+            {
+                var propName = reader.ReadPropertyName();
+                if (string.Compare("NamesToAdd", propName, StringComparison.OrdinalIgnoreCase) == 0)
+                {
+                    namesToAdd = reader.ReadList(JsonReaderExtensions.ReadValueAsString);
+                }
+                else if (string.Compare("NamesToRemove", propName, StringComparison.OrdinalIgnoreCase) == 0)
+                {
+                    namesToRemove = reader.ReadList(JsonReaderExtensions.ReadValueAsString);
+                }
+                else
+                {
+                    reader.SkipPropertyValue();
```

Comment:
this would allow passing additional unused properties?

Reply:
this is generated code from swagger, we arent writing it